src/Stratis.Bitcoin.Features.Api/Controllers/DashboardController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.IO;
+using System.Reflection;
+using System.Text;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Stratis.Bitcoin.Features.Api.Controllers
+{
+    /// <summary>
+    /// Controller serving static pages for the general node dashboard
+    /// </summary>
+    [Route("api/[controller]")]
+    public class DashboardController : Controller
+    {
+        private readonly ILogger logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DashboardController"/> class.
+        /// </summary>
+        public DashboardController(ILoggerFactory loggerFactory)
+        {
+            this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
+        }
+
+        /// <summary>
+        /// Serves up the requested file from the embedded resources
+        /// </summary>
+        /// <returns>Web response with text content</returns>
+        [HttpGet]
+        [Route("static/{file_name}")]
+        public ActionResult Static(string file_name)
+        {
+            var assembly = Assembly.GetEntryAssembly();
+            assembly = Assembly.Load(new AssemblyName("Stratis.Bitcoin.Features.Api"));
```

Comment:
Could this be the assembly of a type you know will be in that assembly rather than loading by string assembly name?  If not, then perhaps a constant for the assembly name